lib/fastlane/plugin/wpmreleasetoolkit/helper/metadata_download_helper.rb

```diff
@@ -58,15 +58,15 @@ def reparse_alternates(target_locale, loc_data, is_source)
             if file[0].to_s == key
               puts "Alternate: #{key}"
```

Comment:
Unrelated to this PR, but I wonder whether we should update this `puts` and the one above at line 57 to `UI.message`?